### PR TITLE
created division feature

### PR DESCRIPTION
**Pull request by pchapl1**
pull request to merge division feature with master

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
+def div_func(a, b):
+	return a / b
+
 if __name__ == '__main__':
 	parser = argparse.ArgumentParser()
 
 	#add arguments here
-	parser.add_argument('--add',mnargs=2, type=int, help='adds two numbers')
+	parser.add_argument('--add', nargs=2, type=int, help='adds two numbers')
+	parser.add_argument('--divide', nargs=2, type=int, help='adds two numbers')
```
Comment (TreySchneider95) [inline calculator.py:15]: change help comment

Comment (pchapl1) [reply calculator.py]: changed help message

**Review by TreySchneider95 — APPROVED** [review]

**Comment by TreySchneider95** [comment]
good work

File: calculator.py [hunk 2]
```diff
@@ -4,13 +4,20 @@
...
 
+def div_func(a, b):
+	return a / b
+
 if __name__ == '__main__':
 	parser = argparse.ArgumentParser()
 
 	#add arguments here
-	parser.add_argument('--add',mnargs=2, type=int, help='adds two numbers')
+	parser.add_argument('--add', nargs=2, type=int, help='adds two numbers')
```
Comment (dd-code-immersives) [inline calculator.py:14]: help messsages needs to be updated

**Review by dd-code-immersives — APPROVED** [review]

**Review by dd-code-immersives — CHANGES_REQUESTED** [review]
make sure b is not 0

**Review by TonyPittella — APPROVED** [review]
cool

**Review by pchapl1 — COMMENTED** [review]
updated division feature to make sure b is not 0

**Review by TreySchneider95 — APPROVED** [review]

**Review by TreySchneider95 — APPROVED** [review]

**Review by dd-code-immersives — APPROVED** [review]

**Pull request merged**